README.md

```diff
@@ -39,20 +50,50 @@ run:
 npx prettier --write ./docs/
 ```
 
+### Static Site Generator
+
+The Wiki's latest version uses the [Docusaurus](https://docusaurus.io/) static website 
+generator to convert the Markdown docs into a documentation website.
+
+### Search Engine
+
+[Algolia DocSearch](https://docsearch.algolia.com/) is the search engine that is used, which 
+is built into Docusaurus. Indexing via Algolia provides faster lookup; the actual configuration
+for lookup is located in another repository that's maintained by Algolia DocSearch.
+
+We have enabled searching on the Wiki by declaring the `algolia` section in the `siteConfig.js` 
+file in `scripts`, and defining an API key and index name that are provided by DocSearch.
+
+```js
+  algolia: {
+    apiKey: "53c6a4ab0d77c0755375a971c9b7cc3d",
+    indexName: "kusama_guide",
+    algoliaOptions: {
+      facetFilters: ["language:LANGUAGE"],
+    }, // Optional, if provided by Algolia
+  }
+```
+
+If you would like to access and modify this, you can re-submit the documentation url via 
+[DocSearch Program](https://docsearch.algolia.com/apply/), where they will send 
+a JavaScript snippet that you can re-integrate into the configuration, similar to the 
+one shown above.
+
 ### Automated Deployments
 
-The wiki is automatically built and deployed via Github Actions for new commits on the
-master branch. The Kusama guide is also deployed to GitHub Pages (via a separate repository). Both
-websites are also uploaded to IPFS via GitHub actions.
+The Polkadot wiki is built on the `gh-pages` branch and automatically deployed to GitHub Pages.
+The Kusama wiki is also deployed to GitHub Pages (via a separate repository). 
 
 ### Dynamic Value Inserts
 
 This documentation sometimes makes references to on-chain values that may change over time. For
 example, it might reference the current number of validators. A custom script exists to populate
 these values post-build. To avoid conflicts in source files, the replacement is done on **built**
-files, not the MD files. The value placeholders are defined in
-[scripts/inject-dict.json](scripts/inject-dict.json). The placeholders should be included in text
-surrounded by double curly braces, like so: `{{ num_validators }}`.
+static files, not the MD files. The value placeholders are defined in the `inject-dict.json` 
+file located in `scripts`. 
+
+The placeholders should be included in text surrounded by double curly braces, 
+like so: `{{ num_validators }}`.
 
 To use the replace script:
```

Comment:
We dont have the scripts for injection anymore

Reply:
Updated